packages/css/src/components/card/card.scss

```diff
@@ -16,12 +15,24 @@
     outline-style: auto;
     outline-width: 0.0625rem;
   }
+
+  & > .ams-heading:has(+ *) {
```

Comment:
Refering to the class of the child components was our initial approach; later in the refinement we decided to add `Card.Heading` and `Card.Image` components to decrease coupling in the stylesheet.

`Card.Heading` will have the props of `Heading`, render a `Heading`, but add a class of `ams-card__heading` on which we set the bottom margin.